content/en/serverless/troubleshooting/_index.md

```diff
@@ -38,35 +38,38 @@ To ensure the actual changes made to your Lambda functions are expected, try set
 
 ## Collect debugging logs
 
-Enable verbose debugging logs by setting the environment variable `DD_LOG_LEVEL` to `debug` on your Lambda functions. If you are using the [Datadog Forwarder Lambda function][2] for data forwarding from logs, also set `DD_LOG_LEVEL` to `debug` on the Forwarder Lambda function. 
+Enable verbose debugging logs by setting the environment variable `DD_LOG_LEVEL` to `debug` on your Lambda functions. If you are using the [Datadog Forwarder Lambda function][2] for data forwarding from logs, also set `DD_LOG_LEVEL` to `debug` on the Forwarder Lambda function.
 
 If you have issues with tracing, set the environment variable `DD_TRACE_DEBUG` to `true` for extra debugging logs from the Datadog Tracer.
 
 To avoid unnecessary costs, disable the debugging logs after you collect enough data.
 
 ## Check the AWS integration
 
-Datadog can collect metrics and resource tags from AWS through an [integration with AWS][10] (optional). If you are missing certain metrics from CloudWatch and Lambda resource tags, ensure that the AWS integration is properly configured. 
+Datadog can collect metrics and resource tags from AWS through an [integration with AWS][10] (optional). If you are missing certain metrics from CloudWatch and Lambda resource tags, ensure that the AWS integration is properly configured.
 
 ## Ensure the tags are configured
 
 If you have trouble applying the Datadog standard `service`, `env`, and `version` tags to the collected data, make sure the environment variables `DD_SERVICE`, `DD_ENV`, and `DD_VERSION` are configured on your Lambda functions. For custom tags, ensure `DD_TAGS` is configured.
 
 If you want to enrich the data collected with your AWS Lambda resource tags, make sure the [Datadog integration for AWS][10] is properly configured.
 
-## Get help
+## Get help with serverless flare
 
 For quick questions, post in the _#serverless_ channel of the [Datadog Slack community][11].
 
-If you have followed all the troubleshooting steps above and want help from [Datadog Support][12], include the following information in your ticket:
+If you have followed all the troubleshooting steps above and want help from [Datadog Support][12], first create a Zendesk ticket. Next, use the `datadog-ci serverless flare` command to automatically collect and submit data about your Lambda function to Datadog Support.
```

Comment:
Should we mention that they need to install `datadog-ci` first?

Reply:
Yes, as someone who is new to Datadog, what even is that command? Where does it come from?
There is no link around where this tool exist, or where to install it.

Reply:
I think there should be a better way of doing this.
We are approaching the customer with 2 ways, either you do it manually, or you let our tool do it. We prefer the first one so it should be the default.

I would recommend following the approach as done in the installation:
https://docs.datadoghq.com/serverless/installation/python/?tab=datadogcli

Reply:
I have updated to follow the approach you recommended ^